### PR TITLE
Improve game_events schema as informed by retrosheet.

### PR DESCRIPTION
**Pull request by Corvimae**
See https://www.retrosheet.org/datause.txt

### CODE REVIEW

File: sibr-cassandra-migrations.cql [hunk 1]
```diff
@@ -103,13 +103,45 @@ CREATE TABLE IF NOT EXISTS blaseball.games(
...
   game_id uuid,
   event_type text,
-  player_id uuid,
-  inning smallInt,
-  numeric_value decimal,
-  string_value text,
-  PRIMARY KEY (game_id, perceived_at)
+  event_index int,
+  inning tinyInt,
+  outs_before_play tinyInt,
```
Comment (tehstone) [inline sibr-cassandra-migrations.cql]: Do we need balls and strikes count as well?

Comment (Corvimae) [reply sibr-cassandra-migrations.cql]: yes I knew I forgot something lol

Comment (tehstone) [reply sibr-cassandra-migrations.cql]: The retrosheet list is significantly longer but many of the field names are rather obscure, can't tell quickly if anything else important is missing.

Comment (Corvimae) [reply sibr-cassandra-migrations.cql]: There's a lot of fielder position fields that we can't track.

File: sibr-cassandra-migrations.cql [hunk 2]
```diff
@@ -103,13 +103,49 @@ CREATE TABLE IF NOT EXISTS blaseball.games(
...
   perceived_at timestamp,
   game_id uuid,
   event_type text,
-  player_id uuid,
-  inning smallInt,
-  numeric_value decimal,
-  string_value text,
-  PRIMARY KEY (game_id, perceived_at)
+  event_index int,
+  inning tinyInt,
```
Comment (tehstone) [inline sibr-cassandra-migrations.cql]: Are we tracking top vs bottom of inning?

Comment (Corvimae) [reply sibr-cassandra-migrations.cql]: I'll add `batter_team` and `pitcher_team`, that should cover it.

Comment (tehstone) [reply sibr-cassandra-migrations.cql]: I guess we can get it from the pitcher or batter id in a roundabout way

**Comment by tehstone** [comment]
Let's add home_strike_count and away_strike_count to events, or strike_count to teams

Let's also add batter_count to teams or to games.

**Comment by Corvimae** [comment]
Need to add `runs_batted_in` too.

**Comment by Corvimae** [comment]
I think the way I modeled players might be too much to keep up, but the problem is storing each stat in a separate `player_stats` table is going to be bad for column store queries. Wondering if maybe it should be `stats: list<{ name: varchar(48); value: double}>` instead?

File: sibr-cassandra-migrations.cql [hunk 3]
```diff
@@ -126,6 +126,9 @@ CREATE TABLE IF NOT EXISTS blaseball.game_events(
   pitcher_team_id uuid,
   home_score smallInt,
   away_score smallInt,
+  home_strike_count smallInt,
+  away_strike_count smallInt,
+  batter_count int,
```
Comment (tehstone) [inline sibr-cassandra-migrations.cql]: This needs to be split into home and away batter counts

**Review by iliana — APPROVED** [review]

**Pull request closed without merging**